ntoskrnl/mm/ARM3/pagfault.c

```diff
@@ -1325,7 +1333,7 @@ MiDispatchFault(IN BOOLEAN StoreInstruction,
             }
 
             /* Resolve the fault -- this will release the PFN lock */
-            Status = MiResolveProtoPteFault(StoreInstruction,
+            Status = MiResolveProtoPteFault(!MI_IS_NOT_PRESENT_FAULT(FaultCode),
```

Comment:
Do you want to mark these with a FIXME or something (since presumably they're meant to check for write instead)?